de.dashup.model/src/test/java/de/dashup/test/LayoutModeTest.java

```diff
@@ -0,0 +1,156 @@
+package de.dashup.test;
+
+import de.dashup.model.db.Database;
+import de.dashup.model.service.DashupService;
+import de.dashup.shared.LayoutModePanel;
+import de.dashup.shared.LayoutModeSection;
+import de.dashup.shared.LayoutModeStructure;
+import de.dashup.shared.User;
+import de.dashup.util.string.Hash;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+class LayoutModeTest {
```

Comment:
This class and all its methods should be public. This might cause some problems, when executing `mvn test`. You can add the class annotation `@SupressWarning("WeakerAccess")`.